python/pyhail/dataset.py

```diff
@@ -251,13 +297,38 @@ def annotate_samples_fam(self, input, quantpheno=False, delimiter='\\\\s+', root
         return self.hc.run_command(self, pargs)
 
     def annotate_samples_list(self, input, root):
-        """Annotate samples with a Boolean indicating presence/absence in a
-        list of samples in a text file.
+        """Annotate samples with a Boolean indicating presence/absence in a list of samples in a text file.
+
+        **Example**
+
+        We have a file `batch1.txt` with a list of samples::
+
+          $ cat data/batch1.txt
+          PT-1234
+          PT-1235
+          PT-1236
+          PT-1237
+          PT-1238
+          PT-1239
+
+        The command
+
+        >>> vds1 = (hc.read('data/example.vds')
+        >>>  .annotate_samples_list('data/batch1.txt','sa.inBatch1')
+
+        will read the file and annotate samples with true if in the list and false otherwise, resulting in the following schema::
+
+          Sample annotations:
+          sa: Struct {
+              inBatch1: Boolean
+          }
```

Comment:
This is a bit misleading, esp. in more complicated examples where there might be other sample annotations.  I'd just say it adds a "sample annotation sa.inBatch1: Boolean" (with appropriate formatting).